app/src/main/res/xml/preferences_synchronization.xml

```diff
@@ -0,0 +1,18 @@
+<?xml version="1.0" encoding="utf-8"?>
+<PreferenceScreen
+        xmlns:android="http://schemas.android.com/apk/res/android"
+        xmlns:search="http://schemas.android.com/apk/com.bytehamster.lib.preferencesearch">
+
```

Comment:
How about an explanation that AntennaPod can sync with different services but you can only enable one of them? If I open the gpodder page, does it say that I need to log out of nextcloud first? Otherwise, the queued actions could get inconsistent when both services don't succeed both.

Reply:
i'll add a radio setting for selecting the service that should be used (or whatever a single choice setting is called) and move the (full-) sync actions right below it.
the preferences for the two services will then only contain login and logout.
ok?

Reply:
How about creating a pop-up on the first login that asks what service you want to use? Then we only need a single synchronization screen.

Reply:
done